functional_tests/android_listener_tests/test_listener_rest.py

```diff
@@ -0,0 +1,168 @@
+import concurrent.futures
+from concurrent.futures import ThreadPoolExecutor
+import os
+
+import lib.settings
+from lib.listener import Listener
+from lib.user import User
+from lib.verify import verify_same_docs
+
+from lib import settings
+import logging
+log = logging.getLogger(settings.LOGGER)
+
+import time
+
+
+def create_listener(target_device, local_port, apk_path, activity, reinstall):
+    return Listener(target_device=target_device, local_port=local_port, apk_path=apk_path, activity=activity, reinstall=reinstall)
+
+
+def test_scenario_two():
```

Comment:
Should have a more descriptive name. Would be nice to have a description about the scenarios being automated.